modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -111,14 +111,55 @@ To delete the SSH connection data to the instance, use <<proc-deleting-connectio
 
 {payg} instance may not be accessible from the {productname} server at all times.
 
-. If the instance exists, but is stopped, the last known credentials will be used to try to connect to the instance. 
+* If the instance exists, but is stopped, the last known credentials will be used to try to connect to the instance. 
     How long the credentials remain valid depends on the cloud provider.
 
-. If the instance no longer exists, but is still registered with SUMA, its credentials are no longer valid and the authentication will fail.
+* If the instance no longer exists, but is still registered with SUMA, its credentials are no longer valid and the authentication will fail.
     The error message is shown in the column Status. 
 +
 [WARNING]
 ====
 The error message only indicates that the instance is not available. 
 Further diagnostics about the status of the instance needs to be done on the cloud provider. 
 ====
+
+
+== Registering {payg} system as a client
+
+You can register a {payg} instance from where you harvest the credentials as a {salt} client.
+
+First set up the {payg} instance to collect authentication data, so it can synchronize the channels.
+
+The rest of the process is the same as for any non-public-cloud client and consists of synchronizing channels, automatic bootstrap scriptcreation, activation key creation and starting the registeration.
+
+For more about registering clients, see xref:client-configuration:registration-overview.adoc[].
+
+[IMPORTANT]
+====
+* The instance needs to have a valid cloud connection registered, otherwise it will not have access to channels.
+
+* If the user removes the cloud packages, the credentials harvesting may stop working.
+
+* Running `registercloudguest --clean` can cause problems if no new cloud connection is registered with the cloud guest command.
+
+* Refreshing or changing the [litera;]``registercloudguest`` connection to the public cloud update infrastructure is fine.  << OM: what does ``fine`` mean?
+====
+
+== {payg} connector process
+
+{payg} connector process uses:
+
+* files stored in [path]``/etc/zypp/credentials`` 
+* certificate that should alredy exist on the machine (instance?). This certificate is added by the cloud infrastructure.
+* an entry in the machine's [literal]``hosts``file that is needed for the cloud plugin to work. It is automatically set by the installed cloud packages.
+
+With this information available, a zypper plugin is invoked to get a valid authentication header.
+(OM: I need a sentence here to bridge the {payg} connector and the authentication header.)
+
+[IMPORTANT]
+====
+* Removing the zypper credentials or the imported certificate,  it (what is IT - the connector?) will stop working.
```

Comment:
This section about "== {payg} connector process" I think can be replaced by a warning or suggestion.
 
```suggestion
Any of the following actions/changes in the payg instance will lead to credentials stopping to work:
* Remove zypper credentials files
* Remove the imported certificates
* Remove cloud-specific entries from `/etc/hosts`
* Stop the service named "??"
```